packages/driver/src/cy/commands/files.ts

```diff
@@ -78,6 +79,9 @@ export default (Commands, Cypress, cy) => {
             contents = Buffer.from(contents)
           }
 
+          // Add the filename as a symbol, in case we need it later (such as when storing an alias)
+          state('current').attributes.fileName = basename(filePath)
```

Comment:
The Command class has a getter/setter for attributes that you could use: https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cypress/command.ts#L15

The attributes property is initialized by the constructor though, so I don't think it's absolutely necessary.

Reply:
Changed over.